### DISCUSSION TITLE
Possible to sort board column by close date?

### DISCUSSION THREAD

**Discussion by Scott-Guinn** [community/community#12559] [Projects and Issues]
Is there anyway to sort a column in the Board layout by the date the issue was closed (or linked PR was merged). 

We have an automated workflow that moves issues to the done column when their linked PRs are merged, and previously when using the non-beta projects, these issues were automatically organized in the Done column by the date they were merged. This made it very easy to see which issues were recently completed.

Is there any way to do this with the new Projects Beta?

**Comment by zazaian** [comment]
Hi all - just checking in here to see if there are any plans for being able to sort issues based on key timestamps. Would be great to be able to sort issues by closed-at, started-at, or really any other pertinent timestamps etc.

If this were really needed at present, I think you'd have to manipulate custom fields to get some similar outcome, but it still wouldn't work quite as pure timestamps, and would need to be managed manually.

Thanks!

**Comment by OwnInTheFoam** [comment]
bump

Would love to have this feature!

**Comment by PeterBocan** [comment]
The silence from GitHub is deafening. Absolutely crucial functionality, in my opinion.

**Comment by retiutut** [comment]
+1 on this feature

**Comment by im-chabani** [comment]
please add this feature !
very crucial to be ableto sort issues on a project by opening date and by closing dates

**Comment by hagenh** [comment]
+1 on this feature

**Comment by schmouk** [comment]
Absolutely need this feature

**Comment by arl** [comment]
gitlab has this...

**Comment by rohvani** [comment]
Shocked GitHub doesn't have this feature...

**Comment by supairish** [comment]
+1 !!!
